2-mandatory/2-oxygen-levels.js

```diff
@@ -11,7 +11,9 @@
     Some string methods that might help you here are .replace() and .substring().
 */
 
-function findSafeOxygenLevel() {}
+function findSafeOxygenLevel(oxygenLevel) {
+  return oxygenLevel.find(value => value > 19.5 <= 23.5);  
```

Comment:
`value > 19.5 <= 23.5` should be split into `value > 19.5 && value <= 23.5`. Don't forgot the other checks too